src/components/vm/disks/diskAdd.jsx

```diff
@@ -358,7 +358,9 @@ export class AddDiskModalBody extends React.Component {
         super(props);
         this.state = {
             validate: false,
-            dialogLoading: true
+            dialogLoading: true,
+            customDiskVerified: false,
```

Comment:
Yeah, as I described more in this comment, since customDiskVerified is set to false by default, the "Add" button is disabled immediately as you open the dialog.